src/updateChangelog.ts

```diff
@@ -91,44 +99,51 @@ function getAllLoggedPrNumbers(changelog) {
   return prNumbersWithChangelogEntries;
 }
 
-async function getCommitHashesInRange(commitRange, rootDirectory) {
+async function getCommitHashesInRange(
+  commitRange: string,
+  rootDirectory?: string,
+) {
   const revListArgs = ['rev-list', commitRange];
   if (rootDirectory) {
     revListArgs.push(rootDirectory);
   }
   return await runCommand('git', revListArgs);
 }
 
-/**
- * @typedef {import('./constants.js').Version} Version
- */
+export interface UpdateChangelogOptions {
+  changelogContent: string;
+  currentVersion?: Version;
+  repoUrl: string;
+  isReleaseCandidate: boolean;
+  projectRootDirectory?: string;
+}
 
 /**
  * Update a changelog with any commits made since the last release. Commits for
  * PRs that are already included in the changelog are omitted.
- * @param {Object} options
- * @param {string} options.changelogContent - The current changelog
- * @param {Version} [options.currentVersion] - The current version. Required if
- *   `isReleaseCandidate` is set, but optional otherwise.
- * @param {string} options.repoUrl - The GitHub repository URL for the current
- *   project.
- * @param {boolean} options.isReleaseCandidate - Denotes whether the current
- *   project is in the midst of release preparation or not. If this is set, any
- *   new changes are listed under the current release header. Otherwise, they
- *   are listed under the 'Unreleased' section.
- * @param {string} [options.projectRootDirectory] - The root project directory,
- *   used to filter results from various git commands. This path is assumed to
- *   be either absolute, or relative to the current directory. Defaults to the
- *   root of the current git repository.
- * @returns {string} The updated changelog text
+ * @param options
+ * @param options.changelogContent - The current changelog
+ * @param options.currentVersion - The current version. Required if
+ * `isReleaseCandidate` is set, but optional otherwise.
+ * @param options.repoUrl - The GitHub repository URL for the current project.
+ * project.
+ * @param options.isReleaseCandidate - Denotes whether the current project.
+ * is in the midst of release preparation or not. If this is set, any new
+ * changes are listed under the current release header. Otherwise, they are
+ * are listed under the 'Unreleased' section.
```

Comment:
`are` repeated twice:
```suggestion
 * listed under the 'Unreleased' section.
```